### PR TITLE
Use comments & simplify issue template

### PR DESCRIPTION
**Pull request by xerus2000**
Filling it out was tedious because one had to replace the instructions rather than simply adding.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
+<!--
 If you encounter a bug or you have a support question, please try to fill out some of the information below.
-However, if you think your issue does not need any of it, you may omit it.
-Generally speaking, the information below is meant for helping debugging issues
-but they are no prerequisite for opening issues.
-
-### Subject of the issue
-Describe your issue here.
+Generally speaking, the information below is meant to help debugging issues but is no prerequisite for opening an issue.
```
Comment (jneira) [inline .github/ISSUE_TEMPLATE.md:3]: I would keep the `However...` to "allow" users delete entirely the template

**Review by jneira — CHANGES_REQUESTED** [review]
Thanks, using the comment feature makes it much more usable

**Review by fendor — APPROVED** [review]
Nice, thank you!

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (xerus2000) [reply .github/ISSUE_TEMPLATE.md:3]: I think the users that want to delete the template will do it anyways ;)
And saying it is "no prerequisite for opening an issue" basically implies that. I'd err on the side of nudging people to use the template.

Comment (jneira) [reply .github/ISSUE_TEMPLATE.md:3]: Fair enough, thanks again for improve it

**Review by jneira — APPROVED** [review]

**Pull request merged**